### PR TITLE
Created mobile middle section (projects and about-me)

### PR DESCRIPTION
**Pull request by Zainab-Alzubair**
In this PR:

created projects section HTML code
styled projects section 
created about-me section 
added style to the about-me section 
added images to img folder 

Thank you for reviewing my PR

### CODE REVIEW

**Review by frank1738 — CHANGES_REQUESTED** [review]
sorry for the delay 👍

File: index.html [hunk 1]
```diff
@@ -41,5 +41,149 @@ <h3>I’am a Software Developer</h3>
               <li class="social"><a href="#"><img src="img\medium.png"></a></li>
             </ul>
       </section>
+      
+      <!--Second section-MY WORK-->
+      <section class="middle">
+        <h1 class="headline">My recent works</h1>
+        <hr>
```
Comment (frank1738) [inline index.html:48]: - Kindly use the correct font style, image sizes, and spacing of elements on your project section to match the Figma design.
 ### Your Design

image

image

### Figma Design

image

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Zainab-Alzubair , 

Exellent work on your project so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Descriptive PR👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Zainab-Alzubair) [reply index.html:52]: Hi @frank1738 
Thank you for your feedback 

I've pushed the changes, Kindly review them 

regards

Comment (frank1738) [reply index.html:52]: Good work @Zainab-Alzubair . Kindly submit your project on the dashboard for another review.

Comment (Zainab-Alzubair) [reply index.html:52]: Alright, thanks @frank1738

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Zainab-Alzubair,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

- [OPTIONAL] Kindly edit the PR title, it can be "Portfolio about section PR".

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Zainab-Alzubair** [comment]
Thank you @Kossi-stack

**Pull request merged**